tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -1016,7 +1016,7 @@ void HWCommandQueue::finish() {
             } else if (tt::llrt::watcher_server_killed_due_to_error()) {
                 this->exit_condition = true;
                 TT_THROW(
-                    "Command Queue could not finish: device hang due to illegal NoC transaction. See build/watcher.log "
+                    "Command Queue could not finish: device hang due to illegal NoC transaction. See built/watcher.log "
```

Comment:
haha, nice

Reply:
lets define the path in watcher.h in case we move it again...

Reply:
Fixed!